storybook/src/foundation/design-tokens/typography.docs.mdx

```diff
@@ -0,0 +1,292 @@
+| `var(--ams-typography-body-text-font-size)`         | `clamp(1.125rem, calc(1.0313rem + 0.4688vw), 1.5rem)`     | <div className="ams-docs-token-example--font-size" style={{ fontSize: 'var(--ams-typography-body-text-font-size)' }}>Abc</div>         |
+| `var(--ams-typography-body-text-large-font-size)`   | `clamp(1.3125rem, calc(1.1719rem + 0.7031vw), 1.875rem)`  | <div className="ams-docs-token-example--font-size" style={{ fontSize: 'var(--ams-typography-body-text-large-font-size)' }}>Abc</div>   |
+| `var(--ams-typography-body-text-x-large-font-size)` | `clamp(1.5313rem, calc(1.3281rem + 1.0156vw), 2.3438rem)` | <div className="ams-docs-token-example--font-size" style={{ fontSize: 'var(--ams-typography-body-text-x-large-font-size)' }}>Abc</div> |
+
+### Line height
+
+The line heights have been designed to match the corresponding font size.
+Do not mix tokens from different variations of body text.
+
+| CSS variable                                          | Value                                         |
+| :---------------------------------------------------- | :-------------------------------------------- |
+| `var(--ams-typography-body-text-small-line-height)`   | `var(--ams-typography-body-text-line-height)` |
+| `var(--ams-typography-body-text-line-height)`         | `1.6`                                         |
+| `var(--ams-typography-body-text-large-line-height)`   | `1.5`                                         |
+| `var(--ams-typography-body-text-x-large-line-height)` | `1.3`                                         |
+
+### Font weight
+
+The font weight can either be regular or bold.
+
+| CSS variable                                       | Value |
+| :------------------------------------------------- | :---- |
+| `var(--ams-typography-body-text-font-weight)`      | `400` |
+| `var(--ams-typography-body-text-bold-font-weight)` | `800` |
+
+## Headings
+
+Headings use a distinct group of tokens because they have distinct levels, and their line heights differ from those for body text.
+
+### Font size
+
+| CSS variable                                | Value                                                     | Example                                                                                                                        |
+| :------------------------------------------ | :-------------------------------------------------------- | :----------------------------------------------------------------------------------------------------------------------------- |
+| `var(--ams-typography-heading-0-font-size)` | `clamp(2.4316rem, calc(1.8951rem + 2.6826vw), 4.5776rem)` | <div className="ams-docs-token-example--font-size" style={{ fontSize: 'var(--ams-typography-heading-0-font-size)' }}>Abc</div> |
```

Comment:
Do we want to make the examples bold? To more accurately show what the text will look like?

Reply:
Absolutely.